core/src/main/java/com/microsoft/applicationinsights/internal/util/PropertyHelper.java

```diff
@@ -28,8 +30,9 @@
 /**
  * Helper class for reading data from a properties file found on the class path.
  */
-public class PropertyHelper
-{
+public final class PropertyHelper {
+    public final static String SDK_VERSION_FILE_NAME = "sdk-version.properties";
```

Comment:
The way I see it, PropertyHelper is just a general helper class to help other classed loading custom property files, and should contain specific files inside it. Otherwise, what makes the sdk-version file different then the other properties files it gets special treatment and its own method while the other files do not? 
Also I wouldn't want to extend this file for every properties file I add to the project.